src/main/java/uk/gov/hmcts/darts/task/config/RemoveDuplicatedEventsAutomatedTaskConfig.java

```diff
@@ -1,13 +0,0 @@
-package uk.gov.hmcts.darts.task.config;
-
-import lombok.Getter;
-import lombok.Setter;
-import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
-
-@ConfigurationProperties("darts.automated.task.remove-duplicated-events")
```

Comment:
Also eliminate the now redundant `remove-duplicated-events` config in `application.yaml`

Reply:
Good spot removed :)